online-bookstore/src/main/java/com/example/onlinebookstore/services/book/BookServiceImpl.java

```diff
@@ -125,4 +132,20 @@ private void updateBookFields(BookDTO bookDTO, BookEntity editedBook) {
         editedBook.setQrCode(bookDTO.getQrCode());
         editedBook.setInventory(bookDTO.getInventory());
     }
+
+    private GenreEntity checkIfGenreExistsInDBAndMerge(GenreEntity genreToBeSaved) {
+        List<GenreEntity> iterateRepositoryList = genreRepository.findAll();
```

Comment:
Variables should not have verbs in the name because don't perform actions. In our case, we can have maybe `genreEntities` or `genreEntityList`.

Reply:
Will be altered, at some point it was confusing to build the logic so my brain was on low power. Had to remeber some variables easily.